sky/skylet/providers/gcp/node_provider.py

```diff
@@ -286,6 +286,9 @@ def get_order_key(node):
                     count -= len(reuse_node_ids)
             if count:
                 results = resource.create_instances(base_config, labels, count)
+                if "sourceMachineImage" in base_config:
+                    for _, instance_id in results:
```

Comment:
Have you tested on multi-node cluster yet?

Reply:
Certainly, I've tested with multi-node cluster with `sky launch -c my-cluster --num-nodes 8 --cloud gcp --disk-size 200`, and ensured that the disk has been resized to 200GB for each node.

Reply:
It'll be great if we add all tests to the PR description ;)

Reply:
Done :)